pki/src/test/resources/cms/crl.pem

```diff
@@ -0,0 +1,53 @@
+This pem file contains the CRL rules for CAs
+- ca
+- interca
+- partneraca
+- selfsigned
+
+They are all empty but partneraca's that contains the revoked certificate used in the tests.
+
+-----BEGIN X509 CRL-----
+MIIBiDByAgEBMA0GCSqGSIb3DQEBCwUAMA0xCzAJBgNVBAMTAmNhFw0yMTA1MzEw
```

Comment:
I hope you have made sure that the CRLs do not expire within the next 100 years?

Reply:
999 years :D